### DISCUSSION TITLE
Triggering link:click twice seems to interfere with content:replace and animation:in

### DISCUSSION THREAD

**Discussion by VisiaMedia** [swup/swup#790] [Help]
Hi there,

First off, thank you for the amazing library and extensive documentation. Implementing Swup into my project has been quite a ride, but I'm loving it so far.

After upgrading V3 to V4, I've been running into some unexpected behavior. Which I suspect is caused by the JS Plugin:

Navigating between pages works fine in most cases, but when (accidentally) double-clicking, the second link:click event seems to stop the expected sequence of events, which causes animation:in to not fire at all. content:replace then won't fire and the transition stays stuck between the out and in stages.

I installed the Debug Plugin to find out what exactly happens. Here's what I ran in to:

**Regular link:click event:**
<img width="245" alt="Screenshot 2023-09-21 at 16 45 44" src="https://github.com/swup/swup/assets/7672294/bcf38432-00dd-4899-87bb-4198267eb82c">

**Double-clicking:**
<img width="192" alt="Screenshot 2023-09-21 at 16 45 59" src="https://github.com/swup/swup/assets/7672294/c024f3e5-103d-44d0-9968-92cbd863f055">

I'm not 100% sure if this behavior is caused by the JS Plugin, and at this point my project relies too much on it to remove the plugin. As my project is quite large, I can't extract a minimal demo. But I do have a build which you can check out [here](https://visiamedia.netlify.app/).

I hope someone can point me into the right direction. Thanks in advance!

**Comment by daun** [comment]
Closing as duplicate/original of issue #791 which has been fixed in release 4.4.1.
